lib/stripe/error_object.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
```

Comment:
Not a strong opinion on this, but do you think we should just stick this in `stripe_error.rb` to keep all the error stuff together?

Reply:
Hm, I'd rather keep it in its own file since it's an API resource of sorts and might be codegen'd at some point.